UnitsNet/CustomCode/Quantities/RotationalStiffness.extra.cs

```diff
@@ -37,6 +37,16 @@ public partial struct RotationalStiffness
         {
             return Torque.FromNewtonMeters(rotationalStiffness.NewtonMetersPerRadian * angle.Radians);
         }
+
+        public static RotationalStiffnessPerLength operator /(RotationalStiffness rotationalStiffness, Length length)
+        {
+            return RotationalStiffnessPerLength.FromNewtonMetersPerRadianPerMeter(rotationalStiffness.NewtonMetersPerRadian / length.Meters);
+        }
+
+        public static Length operator /(RotationalStiffness rotationalStiffness, RotationalStiffnessPerLength rotationalStiffnessPerLength)
```

Comment:
You could later add `RotStiff = RotStiffPerLength * Length` and `RotStiff = Length * RotStiffPerLength` if you find a need for those.

Reply:
Nevermind, I see you already added one of these below :-)